### DISCUSSION TITLE
Webhook to Binance?

### DISCUSSION THREAD

**Discussion by amigolife** [FabCx/TradingView-Webhook-Bot#140] [General]
I just discovered this and it looks very promising!

I use both Telegram and Discord and have TradingView send messages directly to both via webhook.
I can now also send direct orders to Binance for execution.

I haven't tried yet, but just wanted to quickly ask - am I able to send to Binance as well?

**Comment by fabston** [comment]
My script does not offer binance trade execution however you could easily implement that yourself using CCXT.

**Reply by amigolife** [reply]
thanks for the input - much appreciated!
